zokrates_core/src/imports.rs

```diff
@@ -156,253 +68,155 @@ impl Importer {
-                                None => {
-                                    let source = arena.alloc(source);
-
-                                    let compiled = compile_module::<T, E>(
-                                        source,
-                                        new_location.clone(),
-                                        resolver,
-                                        modules,
-                                        &arena,
-                                    )?;
-
-                                    assert!(modules
-                                        .insert(new_location.clone(), compiled)
-                                        .is_none());
-                                }
-                            };
+                "unpack" => SymbolDeclaration {
+                    id: symbol.get_alias(),
+                    symbol: Symbol::Flat(FlatEmbed::Unpack),
+                },
+                "u64_to_bits" => SymbolDeclaration {
+                    id: symbol.get_alias(),
+                    symbol: Symbol::Flat(FlatEmbed::U64ToBits),
+                },
+                "u32_to_bits" => SymbolDeclaration {
+                    id: symbol.get_alias(),
+                    symbol: Symbol::Flat(FlatEmbed::U32ToBits),
+                },
+                "u16_to_bits" => SymbolDeclaration {
+                    id: symbol.get_alias(),
+                    symbol: Symbol::Flat(FlatEmbed::U16ToBits),
+                },
+                "u8_to_bits" => SymbolDeclaration {
+                    id: symbol.get_alias(),
+                    symbol: Symbol::Flat(FlatEmbed::U8ToBits),
+                },
+                "u64_from_bits" => SymbolDeclaration {
+                    id: symbol.get_alias(),
+                    symbol: Symbol::Flat(FlatEmbed::U64FromBits),
+                },
+                "u32_from_bits" => SymbolDeclaration {
+                    id: symbol.get_alias(),
+                    symbol: Symbol::Flat(FlatEmbed::U32FromBits),
+                },
+                "u16_from_bits" => SymbolDeclaration {
+                    id: symbol.get_alias(),
+                    symbol: Symbol::Flat(FlatEmbed::U16FromBits),
+                },
+                "u8_from_bits" => SymbolDeclaration {
+                    id: symbol.get_alias(),
+                    symbol: Symbol::Flat(FlatEmbed::U8FromBits),
+                },
+                s => {
+                    return Err(CompileErrorInner::ImportError(
+                        Error::new(format!("Embed {} not found", s)).with_pos(Some(pos)),
+                    )
+                    .in_file(location)
+                    .into());
+                }
+            },
+            _ => match resolver {
+                Some(res) => match res.resolve(location.to_path_buf(), module_id.to_path_buf()) {
+                    Ok((source, new_location)) => {
+                        let alias = symbol.alias.unwrap_or(
```

Comment:
Cf previous comment, isn't the `unwrap_or` unreachable here as we already determined the default alias? Then how do we determine the default alias if not here?